ui/pages/confirmations/components/confirm/scroll-to-bottom/scroll-to-bottom.tsx

```diff
@@ -49,6 +51,13 @@ const ScrollToBottom = ({ children }: ContentProps) => {
     offsetPxFromBottom: 0,
   });
 
+  const isTransactionRedesign = REDESIGN_USER_TRANSACTION_TYPES.includes(
```

Comment:
Maybe a dumb question, should this to be only check `REDESIGN_USER_TRANSACTION_TYPES` but also if user opted-in for redesigned transactions?

Reply:
I assumed `ScrollToBottom` is used only for redesigned confirmations. I will double-check.

Reply:
That make sense if that's the case

Reply:
In this case we probably want to use `REDESIGN_DEV_TRANSACTION_TYPES` instead, which is a superset of `REDESIGN_USER_TRANSACTION_TYPES` and includes any types not yet visible to the user but already in active development